index.html

```diff
@@ -1374,6 +1593,44 @@ <h3 id="lifetime-AudioContext" class="informative">
             be able to observe garbage collections.
           </p>
         </section>
+        <section>
+          <h3>
+            System resources associated with an AudioContext
+          </h3>
+          <p>
+            <a>AudioContext</a>s should be considered expensive objects.
+            Creating an <a>AudioContext</a> often involves creating an
+            high-priority thread, and use a low-latency system audio stream,
+            both having impact on energy consumption. Creating more than one
+            <a>AudioContext</a> in a document is most of the time unnecessary.
```

Comment:
grammar: "creating an high-priority" to "creating a high-priority"
grammar: "and use a low-latency" to "and using a low-latency"
grammar: "both having impact on" to "both having an impact on"